gradle/scripts/lib/java-publish.gradle

```diff
@@ -129,11 +129,8 @@ configure(projectsWithFlags('publish')) {
         }
     }
 
-    // A version catalog need to publish Gradle metadata together to be correctly imported externally.
-    if (!hasFlags('version-catalog')) {
-        // For now, disable Gradle metadata generation until verifying it is correctly generated.
-        tasks.withType(GenerateModuleMetadata) {
-            enabled = false
-        }
+    // For now, disable Gradle metadata generation until verifying it is correctly generated.
+    tasks.withType(GenerateModuleMetadata) {
+        enabled = false
     }
```

Comment:
Oops, it should not be reverted. Maybe I missed committing the change to `gradle-scripts`. Let me update `gradle-scripts` first.

Reply:
Updated. 😉